docs/tests/drupal.bats

```diff
@@ -0,0 +1,144 @@
+  run ddev drush site:install --account-name=admin --account-pass=admin -y
+  assert_success
+  # ddev launch
+  run bash -c "DDEV_DEBUG=true ddev launch"
+  assert_output "FULLURL https://${PROJNAME}.ddev.site"
+  assert_success
+  # validate running project
+  run curl -sfI https://${PROJNAME}.ddev.site
+  assert_success
+  assert_output --partial "x-generator: Drupal 11 (https://www.drupal.org)"
+  assert_output --partial "HTTP/2 200"
+}
+
+@test "Drupal 10 quickstart with $(ddev --version)" {
+  # mkdir my-drupal-site && cd my-drupal-site
+  run mkdir my-drupal-site && cd my-drupal-site
+  assert_success
+  # ddev config --project-type=drupal10 --docroot=web
+  run ddev config --project-type=drupal10 --docroot=web
+  assert_success
+  # ddev composer create drupal/recommended-project:^10
+  run ddev composer create drupal/recommended-project:^10
+  assert_success
+  # ddev composer require drush/drush
+  run ddev composer require drush/drush
+  assert_success
+  #ddev drush site:install --account-name=admin --account-pass=admin -y
+  run ddev drush site:install --account-name=admin --account-pass=admin -y
+  assert_success
+  # ddev launch
+  run bash -c "DDEV_DEBUG=true ddev launch"
+  assert_output "FULLURL https://${PROJNAME}.ddev.site"
+  assert_success
+  # validate running project
+  run curl -sfI https://${PROJNAME}.ddev.site
+  assert_success
+  assert_output --partial "x-generator: Drupal 10 (https://www.drupal.org)"
+  assert_output --partial "HTTP/2 200"
+}
+# Test for the git based quickstart. Has to be uncommented
+# as soon as the Drupal test repo is created on GitHub
+#@test "Drupal 11 git based quickstart with $(ddev --version)" {
+#  # mkdir my-drupal-site && cd my-drupal-site
+#  run mkdir my-drupal-site && cd my-drupal-site
+#  assert_success
+#  # PROJECT_GIT_URL=https://github.com/ddev/test-drupal.git
+#  PROJECT_GIT_URL=https://github.com/ddev/test-drupal.git
+#  # git clone ${PROJECT_GIT_URL} my-backdrop-site
+#  run git clone ${PROJECT_GIT_URL} .
+#  assert_success
+#  # ddev config --project-type=drupal10 --docroot=web
+#  run ddev config
+#  assert_success
+#  # ddev composer create drupal/recommended-project:^10
+#  run ddev composer install
+#  assert_success
+#  # ddev launch
+#  run bash -c "DDEV_DEBUG=true ddev launch"
+#  assert_output "FULLURL https://${PROJNAME}.ddev.site"
+#  assert_success
+#  # validate running project
+#  run curl -sfI https://${PROJNAME}.ddev.site
+#  assert_success
+#  assert_output --partial "x-generator: Drupal 10 (https://www.drupal.org)"
+#  assert_output --partial "HTTP/2 200"
+#}
+
+@test "Drupal CMS composer quickstart with $(ddev --version)" {
+  # mkdir my-drupal-site && cd my-drupal-site
+  run mkdir my-drupal-site && cd my-drupal-site
+  assert_success
+  # ddev config --project-type=drupal11 --docroot=web
+  run ddev config --project-type=drupal11 --docroot=web
+  assert_success
+  # ddev composer create --stability="RC" drupal/cms
```

Comment:
Please don't make tests different from the quickstart. They should always be the same as the quickstart, at least if possible. Here you're embedding the "RC" here, but not using that in the actual quickstart. I'd prefer to make the quickstart use "RC" until the real thing is ready, then update both the quickstart and the test. But the bottom line is that we want the test to test the *actual* quickstart.